build.gradle

```diff
@@ -1,78 +1,99 @@
-apply plugin: 'com.github.ben-manes.versions'
+//apply plugin: 'com.github.ben-manes.versions'
 
 buildscript {
-  ext.versions = [
-      'minSdk': 9,
-      'compileSdk': 28,
+    ext.versions = [
```

Comment:
**Always** keep the original code style. None of this is supposed to pollute the changelog.

Reply:
#338